<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerGpuWin</class>
 <widget class="QWidget" name="ServerGpuWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel[type=&quot;value&quot;]{
	font: 75 11pt &quot;微软雅黑&quot;;
	color: blue;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtIp">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>172.16.20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>PORT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPort">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>8888</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>GPU ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbGpuId"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>GPU NAME</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lbName">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TEMP(℃)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="lbTemp">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>FREE(M)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lbFree">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>USED(M)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lbUsed">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TOTAL(M)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="lbTotal">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>USED RATE(%)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="lbRate">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>UPDATE TIME</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="lbUpdateTime">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>pid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>显存(M)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>内存(M)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CPU(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>进程</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
